token-metadata/js/README.md

```diff
@@ -0,0 +1,61 @@
+# `@solana/spl-token-metadata`
+
+A TypeScript interface describing the instructions required for a program to implement to be considered a "token-metadata" program for SPL token mints. The interface can be implemented by any program.
+
+## Links
+
+- [TypeScript Docs](https://solana-labs.github.io/solana-program-library/token/js/)
+- [FAQs (Frequently Asked Questions)](#faqs)
+- [Install](#install)
+- [Build from Source](#build-from-source)
+
+## FAQs
+
+### How can I get support?
+
+Please ask questions in the Solana Stack Exchange: https://solana.stackexchange.com/
+
+If you've found a bug or you'd like to request a feature, please
+[open an issue](https://github.com/solana-labs/solana-program-library/issues/new).
+
+## Install
+
+```shell
+npm install --save @solana/spl-token-metadata @solana/web3.js
```

Comment:
If the dependency tree is set up right, they shouldn't have to install both. Odds are they probably have `@solana/web3.js` already, and if they don't need it, our lib will bring only what it needs from `@solana/web3.js` when installed.

Reply:
I don't believe it behaves that way. `@solana/web3.js` is listed as a peer dependency, so it won't get installed when consumers of the library install this package and it will need to be installed. If you already have "@solana/web3.js" installed its safe to install it again - so i think it just serves as a good reminder that its a peer dependency and helps people setting for the first time.

I think this should be safe/preferred and its the same as this package https://github.com/solana-labs/solana-program-library/blob/master/token/js/README.md.

I believe its also standard practices to set it as a peer dependency, so the library version doesn't conflict with installed versions in the consumers code (it can be quite annoying to fix these!). 



Can read more here:
https://www.solutelabs.com/blog/peer-dependencies-in-npm